docs/deploy-configure.md

```diff
@@ -24,12 +24,60 @@ To deploy your first replica set:
    ```
    kubectl get mongodbcommunity --namespace <my-namespace>
    ```
-4. Connect clients to the MongoDB replica set:
+
+4. The Community Kubernetes Operator creates secrets that contains user's connection strings and credentials.
+
+   The secrets follow this naming convention: `<metadata.name>-<auth-db>-<username>`, where:
+
+   Where:
+
+   | Variable | Description | Value in Sample |
+   |----|----|----|
+   | `<metadata.name>` | Name of the MongoDB database resource. | `example-mongodb` |
+   | `<auth-db>` | [Authentication database](https://docs.mongodb.com/manual/core/security-users/#std-label-user-authentication-database) where you defined the database user. | `admin` |
+   | `<username>` | Username of the database user. | `my-user` |
+
+   Update the variables in the following command, then run it to retrieve a user's connection strings to the replica set from the secret:
+
+   **NOTE**: The following command requires `[jq](https://stedolan.github.io/jq/)` version 1.6 or higher.</br></br>
```

Comment:
This hyperlink is not rendering properly.  Perhaps because it is in a code block?